### PR TITLE
AMQ to REST jenkins fix

### PR DESCRIPTION
**Pull request by mkralik3**
//test: `@integrations-amq-to-rest`

### CODE REVIEW

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: ui-tests/src/test/java/io/syndesis/qe/pages/SyndesisPageObject.java [hunk 1]
```diff
@@ -51,8 +51,14 @@ public SelenideElement getButton(String buttonTitle, SelenideElement differentRo
             fail(buttonTitle + " not found", e1);
         }
 
-        return differentRoot.shouldBe(visible).findAll(By.tagName("button"))
-                .filter(Condition.matchText("(\\s*)" + buttonTitle + "(\\s*)")).shouldHave(sizeGreaterThanOrEqual(1)).first();
+        ElementsCollection findedButtons = differentRoot.shouldBe(visible).findAll(By.tagName("button"))
+                .filter(Condition.matchText("(\\s*)" + buttonTitle + "(\\s*)")).shouldHave(sizeGreaterThanOrEqual(1));
+
+        if (findedButtons.size() > 1) {
+            fail("Ambiguous button title. Finded more that 1 button with title " + buttonTitle);
```
Comment (avano) [inline ui-tests/src/test/java/io/syndesis/qe/pages/SyndesisPageObject.java]: `foundButtons`
`Found` 
:dancer:

Comment (mkralik3) [reply ui-tests/src/test/java/io/syndesis/qe/pages/SyndesisPageObject.java]: @avano I have already noticed this reprehensible typo mistake :cry: . I have already fixed it in the local repo.

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: rest-tests/src/test/java/io/syndesis/qe/rest/tests/PublicApiSteps.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package io.syndesis.qe.rest.tests;
+
+public class PublicApiSteps {
+}
```
Comment (avano) [inline rest-tests/src/test/java/io/syndesis/qe/rest/tests/PublicApiSteps.java]: this probably shouldn't be here :grin:

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by avano** [comment]
@mkralik3 ready to merge?

**Comment by mkralik3** [comment]
@avano yep

**Pull request merged**